src/transformers/trainer.py

```diff
@@ -676,43 +676,43 @@ def get_eval_dataloader(self, eval_dataset: Optional[Dataset] = None) -> DataLoa
             pin_memory=self.args.dataloader_pin_memory,
         )
 
-    def get_test_dataloader(self, test_dataset: Dataset) -> DataLoader:
+    def get_predict_dataloader(self, predict_dataset: Dataset) -> DataLoader:
```

Comment:
Changing the Trainer like this is a breaking change that we cannot accept (it will break existing code for users relying on the names of the methods or the names of the arguments). This should not be touched by this PR.

Reply:
Sure @sgugger I will revert these changes